testsuite/features/secondary/proxy_container_cobbler_pxeboot.feature

```diff
@@ -0,0 +1,139 @@
+@proxy
+@private_net
+@pxeboot_minion
+@scope_cobbler
+Feature: PXE boot a terminal with Cobbler and containerized proxy
+  In order to automate client system installations with a containerized proxy
+  As the system administrator
+  I want to PXE boot one host with Cobbler
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+    And I am on the Systems overview page of this "proxy"
+
+  Scenario: Start Cobbler monitoring
+    When I start local monitoring of Cobbler
+
+  # We currently test Cobbler with SLES 15 SP4, even on Uyuni
+  Scenario: Install the TFTP boot package on the server for Cobbler tests
+    When I install package tftpboot-installation on the server
+    And I wait for "tftpboot-installation-SLE-15-SP4-x86_64" to be installed on "server"
+
+  Scenario: Create auto installation distribution
+    When I follow the left menu "Systems > Autoinstallation > Distributions"
+    And I follow "Create Distribution"
+    And I enter "SLE-15-SP4-TFTP" as "label"
+    And I enter "/usr/share/tftpboot-installation/SLE-15-SP4-x86_64/" as "basepath"
+    And I select "SLE-Product-SLES15-SP4-Pool for x86_64" from "channelid"
+    And I select "SUSE Linux Enterprise 15" from "installtype"
+    And I click on "Create Autoinstallable Distribution"
+    Then I should see a "Autoinstallable Distributions" text
+    And I should see a "SLE-15-SP4-TFTP" link
+
+  # WORKAROUND bsc#1195842
+  # Default cobbler kernel parameters are wrong in case of proxy
+  Scenario: Fix kernel parameters
+    When I follow the left menu "Systems > Autoinstallation > Distributions"
+    And I follow "SLE-15-SP4-TFTP"
+    And I enter "useonlinerepo insecure=1 install=http://proxy.example.org/ks/dist/SLE-15-SP4-TFTP self_update=http://proxy.example.org/ks/dist/child/sle15-sp4-installer-updates-x86_64/SLE-15-SP4-TFTP" as "kernelopts"
+    And I click on "Update Autoinstallable Distribution"
+    Then I should see a "Autoinstallable Distribution Updated" text
+
+  Scenario: Create auto installation profile
+    When I follow the left menu "Systems > Autoinstallation > Profiles"
+    And I follow "Upload Kickstart/AutoYaST File"
+    And I enter "15-sp4-cobbler" as "kickstartLabel"
+    And I select "SLE-15-SP4-TFTP" from "kstreeId"
+    And I attach the file "/sle-15-sp4-autoyast.xml" to "fileUpload"
+    And I click on "Create"
+    Then I should see a "Autoinstallation: 15-sp4-cobbler" text
+    And I should see a "Autoinstallation Details" text
+
+  Scenario: Configure auto installation profile
+    When I enter "self_update=0" as "kernel_options"
+    And I click on "Update"
+    And I follow "Variables"
+    And I enter "distrotree=SLE-15-SP4-TFTP\nregistration_key=1-TERMINAL-KEY-x86_64\nredhat_management_server=proxy.example.org" as "variables" text area
+    And I click on "Update Variables"
+    And I follow "Autoinstallation File"
+    Then I should see a "SLE-15-SP4-TFTP" text
+
+  Scenario: Migration of cobbler settings
```

Comment:
The scenario title is a bit misleading, as we use "Given" keyword for "cobbler settings are successfully migrated" it looks like it's a pre-condition. But the title seems to say that the main goal is the migration of the cobbler settings 👀

Reply:
Honestly, I don't even remember why we need it...

There's even a chance it's not needed anymore in containerized setup. I need to ask Ondrej.

Reply:
@aaannz any hint?